cmd/svcat/completion/completion_cmd.go

```diff
@@ -117,3 +119,154 @@ func (c *completionCmd) Run() error {
 func runCompletionBash(w io.Writer, cmd *cobra.Command) error {
 	return cmd.Root().GenBashCompletion(w)
 }
+
+func runCompletionZsh(out io.Writer, cmd *cobra.Command) error {
```

Comment:
Can we replace this function's body with this instead?

```golang
    return cmd.Root().GenZshCompletion(w)
```

Reply:
Turns out that the workaround you have is the best way at the moment, cobra is still working on getting the zsh functionality 100%. 👍

Reply:
I would recommend moving the entire function to it's own file, like `completion_cmd_zsh.go`

Reply:
well, meh... the file only doubles and it is ~250 likes. I take it back.